### DISCUSSION TITLE
Differences vs FluxCD v2

### DISCUSSION THREAD

**Discussion by elgalu** [argoproj/argo-cd#7987] [Q&A] [answered]
Both ArgoCD and Flux2 seem like great solutions, which are the biggest differences? or how to chose one over the other?

**Comment by todaywasawesome** [comment, marked as answer]
@elgalu There is currently an effort to create an in-depth comparison by some members of the GitOps Working Group. That effort is not complete but you can review their [PR here](https://github.com/open-gitops/documents/pull/47/files?short_path=428c188#diff-428c1882b3e2675a269ea1ef323e1de032e68ea47531b0ecf11885de04427615).

My experience with Flux2 is somewhat limited. I'd say some advantages/differences with Argo CD would be:
* Argo CD has a rich UI
* Argo CD is more flexible, projects like [Argo CD Autopilot](https://github.com/argoproj-labs/argocd-autopilot) bring a more opinionated approach, in some ways similar to Flux2. 
* Argo CD uses [GitOps Engine](https://github.com/argoproj/gitops-engine) which is also used and by Gitlab as the core of their GitOps offering.
* Both can be used with Argo Rollouts or Flagger (I believe, I haven't used Flagger but this is my understanding). 
* Argo CD is maintained in a partnership of many companies while Flux is primarily maintained by a single company. 

I'd suggest trying them both and seeing which one you like more.
